weblate/trans/tasks.py

```diff
@@ -67,10 +67,9 @@ def perform_update(cls, pk, auto=False):
             obj = Project.objects.get(pk=pk)
         else:
             obj = Component.objects.get(pk=pk)
-
-        if not auto or settings.AUTO_UPDATE:
+        if settings.AUTO_UPDATE in ("full", True) or auto:
```

Comment:
This inverts the original logic, should be: 
```suggestion
        if settings.AUTO_UPDATE in ("full", True) or not auto:
```